db/migrate/20230207034905_add_is_exclusive_to_lists.rb

```diff
@@ -0,0 +1,6 @@
+class AddIsExclusiveToLists < ActiveRecord::Migration[6.1]
```

Comment:
(also the file name should be updated:

```diff
-db/migrate/20230207034905_add_is_exclusive_to_lists.rb
+db/migrate/20230207034905_add_exclusive_to_lists.rb
```

```suggestion
class AddExclusiveToLists < ActiveRecord::Migration[6.1]
```

Reply:
Ah, missed that. Fixing!